### ISSUE TITLE
Chrome console messages visible in output while running the the tests with logLevel set to silent or error

### ISSUE THREAD

**Issue by MikoSh95** [webdriverio-community/wdio-chromedriver-service#43]
While running tests with WDIO using chromedriver there is Chrome console messages visible in output. There is no way to get rid of them, even by setting webdriver log level to silent.

```
Starting ChromeDriver 88.0.4324.96 (68dba2d8a0b149a1d3afac56fa74648032bcf46b-refs/branch-heads/4324@{#1784}) on port 9515
Only local connections are allowed.
Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.
ChromeDriver was started successfully.
[0-0] RUNNING in chrome - d:\Robocze\bos-frontend-tests\tests\operator\cif\customersCRUDcor.spec.js

DevTools listening on ws://127.0.0.1:53944/devtools/browser/1d076dd7-c8d0-47fc-9c86-6e95d4777b9e

<<<This thing right there>>>
[18196:40464:0202/162948.662:ERROR:device_event_log_impl.cc(211)] [16:29:48.661] USB: usb_service_win.cc:389 Could not read device interface GUIDs: Nie mozna odnalezc okreslonego pliku. (0x2)
[18196:40464:0202/162948.663:ERROR:device_event_log_impl.cc(211)] [16:29:48.662] USB: usb_device_handle_win.cc:1049 Failed to read descriptor from node connection: Urzadzenie dolaczone do komputera nie dziala. (0x1F)
[18196:40464:0202/162948.679:ERROR:device_event_log_impl.cc(211)] [16:29:48.678] USB: usb_device_handle_win.cc:1049 Failed to read descriptor from node connection: Urzadzenie dolaczone do komputera nie dziala. (0x1F)
[18196:40464:0202/162948.679:ERROR:device_event_log_impl.cc(211)] [16:29:48.679] USB: usb_device_handle_win.cc:1049 Failed to read descriptor from node connection: Urzadzenie dolaczone do komputera nie dziala. (0x1F)
```

**Comment by atti187** [comment]
I wouldn't bet on that those lines are console logs from Chrome. They rather look like runtime errors from your OS, or similar? Have you tried googling them?

**Comment by MikoSh95** [comment]
At least some of them are, look here:

```
[0202/204121.339:INFO:CONSOLE(1)] "true true", source: http://10.1.40.82:8080/main.fc6be38ee824826f970e.js (1)
[0202/204121.572:INFO:CONSOLE(1)] "true true", source: http://10.1.40.82:8080/main.fc6be38ee824826f970e.js (1)
[0202/204121.640:INFO:CONSOLE(1)] "true true", source: http://10.1.40.82:8080/main.fc6be38ee824826f970e.js (1)
[0202/204122.002:INFO:CONSOLE(1)] "true true", source: http://10.1.40.82:8080/main.fc6be38ee824826f970e.js (1)
[0202/204124.016:INFO:CONSOLE(73)] "BSSO Telemetry: {"result":"Error","error":"NoExtension","type":"ChromeSsoTelemetry","data":{},"traces":["BrowserSSO Initialized","Creating ChromeBrowserCore provider","Sending message for method CreateProviderAsync","Received message for method CreateProviderAsync","Error: ChromeBrowserCore error NoExtension: Extension is not installed."]}", source: https://login.microsoftonline.com/b2cfc2f5-2191-4c51-ac59-a7f1406b094c/oauth2/authorize?response_type=id_token&client_id=199c207f-e5d2-4c9a-bc5d-a96415db2408&redirect_uri=http%3A%2F%2F10.1.40.82%3A8080%2Fv2&state=d6ba80c9-cf2c-4a5b-8720-ad8791fb2950&client-request-id=5cbc11e8-d694-4fed-b139-a589dd6ae88e&x-client-SKU=Js&x-client-Ver=1.0.18&nonce=54972038-c9c4-404e-ab06-e70a145d3474 (73)
[0202/204124.720:INFO:CONSOLE(73)] "BSSO Telemetry: {"result":"Error","error":"NoExtension","type":"ChromeSsoTelemetry","data":{},"traces":["BrowserSSO Initialized","Creating ChromeBrowserCore provider","Sending message for method CreateProviderAsync","Received message for method CreateProviderAsync","Error: ChromeBrowserCore error NoExtension: Extension is not installed."]}", source: https://login.microsoftonline.com/b2cfc2f5-2191-4c51-ac59-a7f1406b094c/oauth2/authorize?response_type=id_token&client_id=199c207f-e5d2-4c9a-bc5d-a96415db2408&redirect_uri=http%3A%2F%2F10.1.40.82%3A8080%2Fv2&state=bdec4cb3-b93a-45b3-8a76-41a256ac79b0&client-request-id=c4648018-9506-4167-8b74-dbe3d078f5d7&x-client-SKU=Js&x-client-Ver=1.0.18&nonce=32673e68-e90c-453b-80b3-291bc612d9c4 (73)
[0202/204132.532:INFO:CONSOLE(1)] "true true", source: http://10.1.40.82:8080/main.fc6be38ee824826f970e.js (1)
[0202/204132.608:INFO:CONSOLE(1)] "true true", source: http://10.1.40.82:8080/main.fc6be38ee824826f970e.js (1)
[0202/204132.769:INFO:CONSOLE(1)] "true true", source: http://10.1.40.82:8080/main.fc6be38ee824826f970e.js (1)
[0202/204132.947:INFO:CONSOLE(1)] "true true", source: http://10.1.40.82:8080/main.fc6be38ee824826f970e.js (1)
```

First four lines are from our app, I can see this console output when I open this page "manually" in the browser:

![image](https://user-images.githubusercontent.com/23083053/106653708-74dc6500-6597-11eb-99e4-352503f50ef8.png)

I've googled the messages regarding usb devices and it looks like they are also from Chrome console, tak a look here:

https://stackoverflow.com/questions/65080685/usb-usb-device-handle-win-cc1020-failed-to-read-descriptor-from-node-connectio

so I think you might at least take a look if something can be done about it. For couple tests it's not that bad, but now I have more than 300 of them and output gets a little bit messy with those.

**Comment by atti187** [comment]
All this package is doing is to wrap chromedriver up as a service to wdio. It passes the `args` to chromedriver and that's it. Run `./chromedriver --help` to see the set of options that can be used, and elaborate with e.g. `--log-level`.

Another option is to send instructions directly to the browser using the capabilites-configuration and use any of the options listed at https://peter.sh/experiments/chromium-command-line-switches/.

This is as much advice I can give you, as it's not really the responsibility of this package to deal with this.

**Comment by christian-bromann** [comment]
This got fixed with ddbce4845283326f0667f0a122edcf97143da19c
